### DISCUSSION TITLE
springboot/macos-codingmore-run

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#231] [Announcements]
# springboot/macos-codingmore-run

前置环境的准备 一台可以开机的电脑 JDK 8 Maven Intellij IDEA（非必须，可用 Eclipse） Visual Studio Code（非必须，可用 hbuilderx） MySQL Redis（非必须，会报错，但不影响跑） OSS（非必须，上传图片时报错，但不影响跑） 对，编程喵 🐱（Spring Boot+Vue 的前后端分离项目）要想在本地跑起来，需要这 8 个前置环境。

https://tobebetterjavaer.com/springboot/macos-codingmore-run.html

**Comment by xxxxy** [comment]
文中Mac的终端软件是啥？看起来蛮不错的样子
